openshift/topics/get_started.adoc

```diff
@@ -155,45 +179,50 @@ Alternatively, perform one of the following procedures to deploy the {project_na
 
 . Log in to the OpenShift web console and select the _sso-app-demo_ project space.
 
-. On the left sidebar, click the `Administrator` tab and then click `</> Developer`.
+. On the left sidebar, click the *Administrator* tab and then click *</> Developer*.
 +
 image:images/choose_developer_role.png[]
 
-. Click `From Catalog`.
+. Click *From Catalog*.
 +
 image:images/add_from_catalog.png[]
 
-. Search for `sso`.
+. Search for *sso*.
 +
 image:images/sso_keyword.png[]
 
-. Choose a template such as `Red Hat Single Sign-On 7.4 on OpenJDK (Ephemeral)`.
+. Choose a template such as *Red Hat Single Sign-On 7.4 on OpenJDK (Ephemeral)*.
```

Comment:
The 7.4 version should be changed to parametrized form so it's updated each time those paramters are updated.